src/app/testing/gene-mocks.ts

```diff
@@ -17,6 +17,23 @@ export const geneMock1: Gene = {
   is_any_protein_changed_in_ad_brain: true,
   protein_brain_change_studied: true,
   target_nominations: [
+    {
```

Comment:
Added another target nomination to test.  Intentionally chose this one to be out of alphanumeric order.